grails-app/controllers/au/org/ala/specieslist/SpeciesListController.groovy

```diff
@@ -307,14 +328,13 @@ class SpeciesListController {
         }
     }
 
-    def test(){
-        log.debug(helperService.addDataResourceForList("My test List")?.toString())
-        log.debug(authService.getEmail()?.toString())
-        //bieService.bulkLookupSpecies(["urn:lsid:biodiversity.org.au:afd.taxon:31a9b8b8-4e8f-4343-a15f-2ed24e0bf1ae"])
-        log.debug(loggerService.getReasons()?.toString())
-    }
-
```

Comment:
This method has been removed but is still referenced in the file. This is throwing errors.

Reply:
missing method was `getUnmatchedNamesForList`